index.html

```diff
@@ -1,12 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio</title>
-</head>
-<body>
-    
-</body>
-</html>
\ No newline at end of file
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Portfolio</title>
+		<link rel="stylesheet" href="css/style.css" />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+			rel="stylesheet"
+		/>
+	</head>
+	<body>
+		<header class="header container">
+			<div class="header__logo">
+				<h2>My Logo</h2>
+			</div>
+			<nav class="header__nav">
+				<ul>
+					<li><a href="#">Portfolio</a></li>
+					<li><a href="#">About</a></li>
+					<li><a href="#">Contact</a></li>
+				</ul>
+			</nav>
+			<div class="header__menu">
+				<img src="assets/img/Union.svg" alt="menu" />
+			</div>
+		</header>
+		<main>
+			<section class="hero">
+				<div class="hero__main-content">
+					<h1>I’m Alexandra. Glad to see you!</h1>
+					<p>
+						I’m a software developer! I can help you build a product , feature
+						or website Look through some of my work and experience! If you like
+						what you see and have a project you need coded, don’t hesitate to
+						contact me.
+					</p>
+				</div>
+				<div class="hero__connect">
+					<h3>LET'S CONNECT</h3>
+					<div class="hero__connect--social">
+						<img src="assets/img/Twitter.svg" alt="twitter" />
+						<img src="assets/img/Linkedin.svg" alt="linkedin" />
+						<img src="assets/img/Medium.svg" alt="medium" />
+						<img src="assets/img/Github.svg" alt="github" />
+						<img src="assets/img/AngelList.svg" alt="angellist" />
+					</div>
```

Comment:
- [ ] You've done a great job adding icons to your project :+1:. However, it's important to note that the list of icons should be grouped using the `ul` tag, with each icon contained within a `li` element. Additionally, it's necessary to include a link (`a` tag) for each icon to ensure that users can interact with them properly.
Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

Reply:
@IngDavidTM It could be but it is not always necessary to use a ul tag to group icons, although it could be understood that the icons could be anchors, it has not been required any of those characteristics in the requirements of this task. I have not been asked or given URLs corresponding to those icons. So I have decided to opt for code readability. I didn't think it would be a requirement for approval.